telegram/_inline/inlinequery.py

```diff
@@ -43,6 +43,12 @@ class InlineQuery(TelegramObject):
     Note:
         In Python :keyword:`from` is a reserved word use :paramref:`from_user` instead.
 
+    .. versionchanged:: 20.0
+
+        * The following are now keyword-only arguments in Bot methods:
+          ``{read, write, connect, pool}_timeout``, ``api_kwargs``, ``auto_pagination``. Use a named argument
```

Comment:
``` suggestion
          ``{read, write, connect, pool}_timeout``, ``api_kwargs``, :paramref:`answer.auto_pagination`. Use a named argument
```
since its being used in only one place I guess we can directly reference it where